build-deb.Dockerfile

```
@@ -1,9 +1,8 @@
 FROM ubuntu:16.04
-ARG DEB_VERSION=unknown
 WORKDIR /src/
-RUN apt-get update && apt-get install -y curl gcc pkg-config libssl-dev
+RUN apt-get update && apt-get install -y curl gcc pkg-config libssl-dev git
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
 ENV PATH="/root/.cargo/bin:${PATH}"
 RUN cargo install cargo-deb
 COPY . .
-RUN cargo deb -p yagna --deb-version $DEB_VERSION
+ENTRYPOINT cargo deb -p $PACKAGE_NAME --deb-version "$(cargo pkgid $PACKAGE_NAME | awk -F ":" '{print $NF}')-$(git rev-parse --short HEAD)" -- --bins --example ya_sb_router
```

```suggestion
ENTRYPOINT cargo deb -p $PACKAGE_NAME --deb-version "$(cargo pkgid $PACKAGE_NAME | awk -F "#" '{print $NF}')-$(git rev-parse --short HEAD)" -- --bins --example ya_sb_router
```

I dunno how it works @ GH CI, since `cargo pkgid` locally returns
```
awokado:~/git/yagna km/deb-artifact-name (7ac7a16b)$ cargo pkgid
file:///home/tworec/git/yagna#0.3.0
```
 
And this command returns me consequently as below
```
awokado:~/git/yagna release/vm-poc (7091faab)$ cargo deb -p yagna --deb-version "$(cargo pkgid $PACKAGE_NAME | awk -F ":" '{print $NF}')-$(git rev-parse --short HEAD)" -- --bins --example ya_sb_router
warning: extended-description field missing. Using README.md, but markdown may not render well.
   Compiling futures v0.1.29
   Compiling cc v1.0.41

--- CUT FOR BREVITY ---

   Compiling ya-requestor v0.1.0 (/home/tworec/git/yagna/agent/requestor)
   Compiling yagna v0.3.3-alpha.4 (/home/tworec/git/yagna)
    Finished release [optimized] target(s) in 8m 23s
/home/tworec/git/yagna/target/debian/yagna_///home/tworec/git/yagna#0.3.3-alpha.4-7091faab_amd64.deb
```

---

Ok, I've found that within the docer it is other way:
```
root@aae49a6ae6a0:/src# cargo pkgid
file:///src#yagna:0.3.0
root@aae49a6ae6a0:/src# cargo pkgid ya-sb-router
file:///src/service-bus/router#ya-sb-router:0.1.0
```